torchao/quantization/subclass.py

```diff
@@ -832,7 +836,54 @@ def __torch_dispatch__(cls, func, types, args, kwargs):
                     args[1],
                     None if len(args) == 2 else args[2],
                 )
-            if weight_qtensor.input_quant_func is not None:
+            if weight_qtensor.input_quant_func is None:
```

Comment:
Let's please remove `input_quant_func` and discuss using `AffineQuantizedTensor` as an organizing principle :)

Reply:
this is used for testing 8da4w right now, can this be done later until there is a better alternative?